documentation/developer_setup.md

```diff
@@ -310,7 +314,6 @@ func TestClientWithAAD(t *testing.T) {
 
 The `FakeCredential` show here implements the `azcore.TokenCredential` interface and can be used anywhere the `azcore.TokenCredential` is used.
 
-
 ## Create Pipelines
```

Comment:
Do we want to include something on test resources and including an arm or bicep file?

Reply:
Also, do we want to note on any other release requirements? For example, minimum code coverage

Reply:
@benbp do we have any general docs for the repos about arm/bicep files that I can link to here?

Reply:
@jhendrixMSFT We have some internal docs [here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/48/Create-a-new-Live-Test-pipeline), but nothing public aside from official docs. I can take a work item to move some of these docs out into public as they are due for a refresh anyway.

Reply:
That said, it would probably be good to link to [these docs](https://github.com/Azure/azure-sdk-tools/tree/main/eng/common/TestResources) as a starting point, as it shows how to set things up and test locally when building out new live tests.

Reply:
added a section about deploying test resources